src/pages/my-account/users/index.tsx

```diff
@@ -0,0 +1,225 @@
+    return `${mdScreen && t('displaying')} ${startRange} - ${
+      endRange > totalCount ? totalCount : endRange
+    } of ${totalCount}`
+  }
+
+  const confirmDelete = (id: string | undefined | null) => {
+    showModal({
+      Component: ConfirmationDialog,
+      props: {
+        contentText: t('delete-user-confirmation-text'),
+        primaryButtonText: 'Delete',
+        onConfirm: () => {
+          const accountId = user?.id
+          const queryVars = { accountId, userId: id }
+          removeCustomerB2bUser.mutateAsync({ ...queryVars })
+        },
+      },
+    })
+  }
+
+  const AddUserButton = () => {
+    return (
+      <Button
+        variant="primary"
+        disabled={isUserFormOpen}
+        onClick={() => setUserFormOpen(true)}
+        disableElevation
+        id="formOpenButton"
+      >
+        <span style={{ display: 'flex', alignItems: 'center' }}>
+          <AddCircleOutlineIcon style={{ marginRight: '8px', width: '19px' }} />
+          <span style={{ paddingTop: '2px', fontWeight: '400' }}>{t('add-user')}</span>
+        </span>
+      </Button>
+    )
+  }
+
+  return (
+    <Grid>
+      <Grid item style={{ marginTop: '10px', marginBottom: '40px' }}>
+        <BackButtonLink aria-label={t('my-account')} href="/my-account">
+          <ChevronLeftIcon />
+          {mdScreen && <Typography variant="body1">{t('my-account')}</Typography>}
+        </BackButtonLink>
+        <Box sx={style.heading}>
+          <Typography
+            variant={mdScreen ? 'h1' : 'h2'}
+            sx={{ textAlign: { xs: 'center' }, marginTop: { xs: '-3.6rem', md: 0 } }}
+          >
+            {t('users')}
+          </Typography>
+        </Box>
+        <Grid container>
+          <Grid item xs={12} md={12}>
+            {mdScreen && <AddUserButton />}
+            {!mdScreen && !isUserFormOpen && <AddUserButton />}
+            {isUserFormOpen && (
+              <UserForm isEditMode={false} closeUserForm={() => setUserFormOpen(false)} />
+            )}
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid item>
+        <SearchBoxContainer>
+          <SearchBar
+            onSearch={(searchText) =>
+              setPaginationState({
+                ...paginationState,
+                searchTerm: searchText,
+              })
+            }
+            placeHolder={t('user-search-placeholder')}
+            searchTerm={paginationState.searchTerm}
+            showClearButton={true}
+          />
+        </SearchBoxContainer>
+
+        {isLoading ? (
+          <Box style={{ width: '100%', display: 'flex', justifyContent: 'center' }}>
+            <CircularProgress />
+          </Box>
+        ) : (
+          <>
+            <UserTable
+              b2bAccountUsers={data?.items}
+              deleteUser={confirmDelete}
+              editUserId={editUserId}
+              setEditUserId={(id: Maybe<string> | undefined) => setEditUserId(id)}
+            />
+            <PaginationContainer>
```

Comment:
Mui Table provides pagination out of the box. We should use that

Reply:
@kibo-chandradeeptalaha we have used MUI pagination.
Only to show the text at the right side of table we have used Typography.
And this is not provided by MUI lib